cardano-cli/src/Cardano/CLI/EraBased/Query/Run.hs

```diff
@@ -1835,19 +1832,11 @@ runQuerySPOStakeDistribution
               | (keyHash, addr) <- Map.toList $ L.psStakePoolParams poolState
               ]
 
-        mkQueryStakeAddressInfoCmdArgs addr =
-          Cmd.QueryStakeAddressInfoCmdArgs
-            { Cmd.commons = commons
-            , addr
-            , outputFormat
-            , mOutFile -- unused anyway. TODO tighten this by removing the field.
```

Comment:
Both `outputFormat` and `mOutFile` are unused by `callQueryStakeAddressInfoCmd`, so don't bother using `QueryStakeAddressInfoCmdArgs` and packing unnecessary fields.